client-templates/azure-repos/.env.sample

```diff
@@ -29,3 +29,13 @@ BROKER_HEALTHCHECK_PATH=/healthcheck
 
 # Azure validation url, checked by broker client systemcheck endpoint
 BROKER_CLIENT_VALIDATION_URL="https://$AZURE_REPOS_HOST/$AZURE_REPOS_ORG/_apis/git/repositories"
+
```

Comment:
A general question- What about all other integrations that can be brokered? Why are there no `.env` changes for them as well?

Reply:
@ohad2712 We are only targeting git based integrations (so no docker, or jira). I believe I changed `.env` for all of them.